pages/info/info.module.css

```
@@ -0,0 +1,60 @@
+.body {
+    background: rgb(255, 255, 255);
+    background: radial-gradient(circle, rgba(255, 255, 255, 1) 0%, rgba(231, 231, 231, 1) 35%);
+    opacity: 1;
+    min-height: 100vh;
+    padding: 0 0.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+}
```

He hecho este comentario ya a una de tus compañeras. Es posible que sí, pero valdría la pena comprobarlo: siendo que establecéis el height más abajo al mismo valor, ¿es realmente necesario establecer el min-height?

Por otro lado, ¿es necesario establecer la opacity del body a 1?

Y por último, otro detalle:
```css
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
```
Se puede sustituir (en la mayoría de los casos) por:
```css
    display: grid;
    place-items: center;
```

```suggestion
.body {
    background: rgb(255, 255, 255);
    background: radial-gradient(circle, rgba(255, 255, 255, 1) 0%, rgba(231, 231, 231, 1) 35%);
    padding: 0 0.5rem;
    display: grid;
    place-items: center;
    height: 100vh;
```

---

Marcos, gracias por el comentario :)
El Style de .body lo copie de Home.modules.css puntualmente de .container capaz por eso, coincide con el de mi compañera.